src/lib.rs

```diff
@@ -133,6 +133,9 @@ pub type ssize_t = isize;
 pub enum FILE {}
 pub enum fpos_t {} // TODO: fill this out with a struct
 
+#[cfg(not(any(dox, target_env = "msvc")))]
```

Comment:
Ok, I'm just about back in the land of the living after a week long standards love in. I'm happy to add this to `src/unix`, but to add it to `src/windows` would require quite an invasive change - in essence, moving `windows.rs` to `windows/mod.rs` and then introducing `windows/msvc.rs` and `windows/gnu.rs`, say. I'm not completely comfortable doing that for one teensy constant... especially as I don't have the local infrastructure to test my changes thoroughly on windows.